firmware/application/freqman.hpp

```diff
@@ -96,7 +97,7 @@ using freqman_db = std::vector<freqman_entry>;
 
 std::vector<std::string> get_freqman_files();
 bool load_freqman_file(std::string& file_stem, freqman_db& db);
-bool load_freqman_file_ex(std::string& file_stem, freqman_db& db, bool load_freqs, bool load_ranges, bool load_hamradios);
+bool load_freqman_file_ex(std::string& file_stem, freqman_db& db, bool load_freqs, bool load_ranges, bool load_hamradios, uint8_t limit);
```

Comment:
nit: maybe "max_items" or something? "limit" is vague. Also, maybe a default so all the callers don't have to be updated?

(you don't have to fix nits, just suggestions)

Reply:
I changed it to max_num_freqs and gave him the default FREQMAN_MAX_PER_FILE value